### ISSUE TITLE
Translatable link missing in https://www.electronjs.org/docs/latest/api/browser-window

### ISSUE THREAD

**Issue by albu77** [electron/website#357]
In the api/browser-window the link shown on the following picture is pointing to a valid anchor but as there is no hidden text corresponding to this anchor link in the crowdin system the link is not working in the locale version.

![Capture](https://user-images.githubusercontent.com/624802/215139952-8ca977f7-33ba-4340-8078-e9706cf8d14b.PNG)

**Comment by erickzhao** [comment]
@albu77 Does it not work if you change the anchor to `#d%C3%A9finition-de-la-propri%C3%A9t%C3%A9-backgroundcolor`?

**Comment by erickzhao** [comment]
I tried fixing it here: https://crowdin.com/translate/electron/258633/en-fr#7514508

**Comment by albu77** [comment]
Good job, **it's working.**
**But** to my sense, generally speaking  it would be better if we could do in the same manner (the right one for me) than in/docs/latest/tutorial/security. 
as it 's shown below:
![Capture](https://user-images.githubusercontent.com/624802/215260802-66916d08-1893-4630-82cf-dda90c454a84.PNG)
_**And speaking about this one the -7 option (issue #264) is still unsolved.**_
All this stuff around links is a little bit puzzling for me because I have not a general overview of the interaction of docs in the code, docs in the "website", docusaurus etc...

**Comment by erickzhao** [comment]
@albu77 I'm unfamiliar with this part of the translation flow for Crowdin so let me figure out what the best practice for this will be.

**Comment by albu77** [comment]
Yes I think it's interesting if you can figure it because it's a case often met
